xarray/backends/h5netcdf_ng.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
TODO
I didn't do anything in this module yet - this is still using the old API.